### DISCUSSION TITLE
Since this is no longer being actively updated, which fork are you using?

### DISCUSSION THREAD

**Discussion by mechanicmuthu** [imartinez/privateGPT#842] [General]
This privateGPT by imartinez evidently is no longer being updated / enhanced (as of 10th july 2023, last update is almost a month ago.) what fork of privateGPT are you guys using?

**Comment by pseudotensor** [comment]
I'm not sure it's what you are looking for, but h2oGPT has pulled some ideas from privateGPT and shared as well.  https://github.com/h2oai/h2ogpt .

However, privateGPT is great for easy entrance hacking exactly because it's not well-developed, although it's mostly just langchain and other packages put together with a bit of glue.

**Reply by andrewhsandler** [reply]
I don't know anything about any of this im just trying to build myself a second "memory" outside of my head to help me. Anything you recommend?
